locales/de.json

```diff
@@ -740,6 +740,7 @@
 		},
 		"text": {
 			"loginFailed": "Login fehlgeschlagen.",
+			"superheroForbidden": "Superhero forbidden",
```

Comment:
maybe update this into something like 'superheroes may login in superhero dashboard only'

Reply:
OK. I've requested an error message text by the UX/UI team.

Reply:
Proper message texts added (de.json, en.json)